hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/ExplainStatementTest.java

```diff
@@ -236,10 +236,10 @@ public void test_explainStatementInsert() {
                 new Row("InsertPhysicalRel(" +
                         "table=[[hazelcast, public, map[projects=[$0, $1]]]], operation=[INSERT], flattened=[false])"),
                 new Row("  ValuesPhysicalRel(values=[{expressions=[" +
-                        "[ConstantExpression{type=QueryDataType {family=INTEGER}, value=3}, " +
-                        "ConstantExpression{type=QueryDataType {family=INTEGER}, value=3}], " +
-                        "[ConstantExpression{type=QueryDataType {family=INTEGER}, value=4}, " +
-                        "ConstantExpression{type=QueryDataType {family=INTEGER}, value=4}]]}])")
+                        "[ConstantExpression{type=INT, value=3}, " +
```

Comment:
`QueryDataType.toString()` changes EXPLAIN output and logs. Including `"QueryDataType"` was redundant, but I am not sure what is more important: type or family (+precision?). @Fly-Style WDYT?

Reply:
@k-jamroz  The exact type is better than just family.

Reply:
Moved to the upcoming PR.